tests/phpunit/tests/kses.php

```diff
@@ -1596,6 +1596,18 @@ function data_wp_kses_object_tag_allowed() {
 				'<object type="application/pdf" data="/cat/foo.pdf" />',
 				'',
 			),
+			'relative url'                            => array(
+				'<object type="application/pdf" data="/cat/foo.pdf" />',
+				'',
+			),
+			'url with port number'                    => array(
```

Comment:
Not quite sure why these are failing yet.

Reply:
The test suite's host domain is set to use example.org, so these will fail due to a mismatch. 

You may need to create another test/data provider combination and filter the base url using the `upload_dir` hook.

https://github.com/WordPress/wordpress-develop/blob/4ab922d22479074670fb015fd8ceaca70fb5adf7/wp-tests-config-sample.php#L64